README.md

```diff
@@ -80,7 +80,7 @@ Introduction - [slides](http://dask.github.io/dask-tutorial/introduction.html)
 
 4.  Imperative Programming
 
-    *  [Imperative - `do`](04-Imperative.ipynb)
```

Comment:
We've stopped using the term "imperative" as well.  It's not well known by all programmers.  I've started using terms like "generic" or "arbitrary" or "for-loopy" to try to describe this.  Thoughts?